README.md

```diff
@@ -5,24 +5,27 @@ When hosting a BAPC regional one can need:
 - For DOMjudge (https://www.domjudge.org/) 64^2 (PNG)
 - ICPC Resolver (https://tools.icpc.global/resolver/) 256^2 (PNG)
 
-| ICPC ID | Name | Status |
-|---|---|---|
+| ICPC ID | Name | Status             |
+|---|---|--------------------|
 | U-3546 | Ghent University | :heavy_check_mark: |
-| U-3088 | Avans Hogeschool 's-Hertogenbosch| :x: |
+| U-3088 | Avans Hogeschool 's-Hertogenbosch| :x:                |
 | U-0635 | TU Delft | :heavy_check_mark: |
 | U-0711 | TU/e | :heavy_check_mark: |
-| U-3179 | Katholieke Universiteit Leuven | :x: |
+| U-3179 | Katholieke Universiteit Leuven | :heavy_check_mark:         |
 | U-1172 | Leiden University | :heavy_check_mark: |
 | U-7250 | Radboud University | :heavy_check_mark: |
-| ~~U-2096~~ | ~~Universiteit Utrecht~~ | :x:  |
+| ~~U-2096~~ | ~~Universiteit Utrecht~~ | :x:                |
 | U-2097 | Universiteit van Amsterdam | :heavy_check_mark: |
 | U-2307 | University of Twente | :heavy_check_mark: |
-| U-3199 | Université catholique de Louvain | :x: |
+| U-3199 | Université catholique de Louvain | :heavy_check_mark:                |
 | U-7560 | Université de Mons | :heavy_check_mark: |
 | U-7642 | Université Libre de Bruxelles | :heavy_check_mark: |
 | U-2347 | Utrecht University | :heavy_check_mark: |
 | U-2381 | Vrije Universiteit Amsterdam | :heavy_check_mark: |
 | U-10500 | Maastricht University | :heavy_check_mark: |
+| U-7019 | Leiden University of Applied Sciences | :heavy_check_mark:         |
+| U-23553 | University of Luxembourg | :heavy_check_mark:         |
+| U-1529 | University of Groningen | :heavy_check_mark:         |
```

Comment:
I think the universities used to be ordered (mostly) alphabetically. Can we order them again? :slightly_smiling_face: 

(ordering by ICPC ID is also fine if you prefer that, as long as the order is not arbitrary :stuck_out_tongue:)